contracts/src/v0.8/tests/FunctionsBillingRegistryMigration.sol

```diff
@@ -0,0 +1,871 @@
+// SPDX-License-Identifier: MIT
```

Comment:
For reviewing purposes the only difference is the fee on line 287 returns 1 instead of 0.